src/main/java/de/szut/lf8_project/project/ProjectService.java

```diff
@@ -36,5 +36,16 @@ public ProjectEntity readById(long id) {
     public List<ProjectEntity> readAll() {
         return this.repo.findAll();
     }
+
+    public void delete(long id) {
+        ProjectEntity deleteProject = readById(id);
```

Comment:
"readById" already throws a ResourceNotFoundException on failure.
there is no need to check anythin again